src/shared/json-queue.c

```diff
@@ -105,8 +133,260 @@ cJSON * jqueue_next(file_queue * queue) {
     }
 }
 
+/* Re Handle the file queue */
+static int Handle_JQueue(file_queue *fileq, int flags) {
+    /* Close if it is open */
+    if (!(flags & CRALERT_FP_SET)) {
+        if (fileq->fp) {
+            fclose(fileq->fp);
+            fileq->fp = NULL;
+        }
+
+        /*
+            We must be able to open the file, fseek and get the
+            time of change from it.
+        */
+        fileq->fp = fopen(fileq->file_name, "r");
+        if (!fileq->fp) {
+            /* Queue not available */
+            return 0;
+        }
+    }
+
+    /* Seek to the end of the file */
+    if (!(flags & CRALERT_READ_ALL)) {
+        if (!fileq->fp) {
+            return 0;
```

Comment:
Add an error/warning message to know that the file is not available

Reply:
Error message added in line: https://github.com/wazuh/wazuh/pull/4256/files#diff-4389a0705fc67fca0285ad7bf1614e46R155